tekton/release-cheat-sheet.md

```diff
@@ -143,6 +143,7 @@ the pipelines repo, a terminal window and a text editor.
    and push it to the repo https://github.com/tektoncd/pipeline.
    (This can be done on the Github UI.)
    Make sure to fetch the commit specified in `TEKTON_RELEASE_GIT_SHA` to create the released branch.
+   > Background: The reason why we need to create a branch for the release named `release-<version number>x` is for future patch releases. Cherrypicked PRs for the patch release will be merged to this branch. For example, [v0.47.0](https://github.com/tektoncd/pipeline/releases/tag/v0.47.0) has been already released, but later on we found that an important [PR](https://github.com/tektoncd/pipeline/pull/6622) should have been included to that release. Therefore, we need to do a patch release i.e. v0.47.1 by [cherrypicking this PR](https://github.com/tektoncd/pipeline/pull/6622#pullrequestreview-1414804838), which will trigger [tekton-robot to create a new PR](https://github.com/tektoncd/pipeline/pull/6622#issuecomment-1539030091) to merge the changes to the [release-v0.47.x branch](https://github.com/tektoncd/pipeline/tree/release-v0.47.x).
```

Comment:
I think it would be sufficient to say here that we create a release branch in case we need to cherry-pick commits to it later for a patch release-- no need to go into additional details.

If you want to cover cherry-picking, it's probably worth updating our patch release docs on how to cherry-pick with and without the automation.

Reply:
+1, we could document the formal way of cherry picking process, and then give the detailed example